<html lang="en">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta charset="utf-8">
  
    <title>PMEN :: Pneumococcal Molecular Epidemiology Network</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
    <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.1.0/css/bootstrap.min.css">
    <link rel="stylesheet" href="css/normalize.css">
    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400italic,600italic,400,300,600' rel='stylesheet' type='text/css'>
    <!-- fonts, icons for bootstrap -->
    <link href="http://netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.css" rel="stylesheet">
    <link href="css/styles.css" rel="stylesheet">
</head>
<body>

    <!-- Navigation bar starts -->

    <div class="navbar navbar-fixed-top bs-docs-nav top-menu" role="banner">
      <div class="container">
        
        <div class="navbar-header">
          <a href="index.html" class="brand">
            <img src="img/pmen-logo.png">
          </a>
         <button class="navbar-toggle" type="button" data-toggle="collapse" data-target=".bs-navbar-collapse">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
         </button>
        </div>
       

        <nav class="collapse navbar-collapse bs-navbar-collapse" role="navigation">
            <ul class="nav navbar-nav pull-right">
                <li >
                    <a href="index.html">
                        <!-- <i class="fa fa-home blue"> </i> --> Home
                    </a>
                </li>
                <li class="dropdown">
                    <a class="dropdown-toggle" data-toggle="dropdown" href="#">
                         About <b class="caret"></b>
                    </a>
                    <ul class="dropdown-menu">
                        <li class="">
                            <a  href="history.html" >History</a>
                        </li>
                        <li class="">
                            <a href="members.html">Members</a>
                        </li>
                    </ul>
                </li>
                <li class="dropdown active">
                    <a class="dropdown-toggle" data-toggle="dropdown" href="#">Clones <b class="caret"></b></a>
                    <ul class="dropdown-menu">
                        <li class=""><a href="criteria.html">Criteria & Nomenclature</a></li>
                        <li class=""><a href="clone-collection.html">Clone Collection</a></li>
                        <li class=""><a href="map_view.html" >Clone Map</a></li>
                        <li class="active"><a href="molecular-typing.html">Molecular Typing</a></li>
                    </ul>
                </li>
                <li><a href="contact.html"> Contact</a></li>
            </ul>
          </nav>
       </div>
    </div>
<!-- Navigation bar ends -->

<div id="content">    <section>
      <div class="container content justify">
        <header>
            <h1>Molecular Typing</h1>
        </header>
        <div class="desc">
          <p>
            Clones to be included into the Network will be subjected to PFGE, MLST and PBP fingerprinting to confirm that they differ from previously accepted clones.  If macrolide resistant then tests for erm and mef genes will be performed.
          </p>

          <header>
            <h2>Pulsed-field gel electrophoresis (PFGE):</h2>
          </header>
          <p>
            Genomic DNA is prepared in situ in agarose blocks as described previously (<a href="#1">1</a>, <a href="#2">2</a>) and is digested with SmaI (Life Technologies, Gaithersburg Md). The fragments are resolved by PFGE in 1% agarose (SeaKem GTG agarose; BioWhittaker Molecular Applications, Rockland, Maine) in 0.5Xtris-Borate-EDTA buffer at 14Â°C at 6V/cm in a CHEF-DR III system (Bio-Rad Laboratories, Hercules, California). The parameters in block 1 are an initial pulse time of 1s increased to 30s for 17 hrs, and in block 2, 5s increased to 9s for 6 hrs.
          </p>
          <header>
            <h2>Multilocus sequence typing:</h2>
          </header>
          <p>
            MLST is carried out as described previously (<a href="#3">3</a>). Briefly, internal fragments of the aroE, gdh, gki, recP, spi, xpt and ddl genes are amplified by PCR from chromosomal DNA using the primer pairs described by Enright and Spratt (<a href="#3">3</a>). The amplified fragments are directly sequenced in each direction using the primers that are used for the initial amplification. 
          </p>
          <p>
            The sequences at each of the seven loci are then compared with all of the known alleles at that locus. Sequences that are identical to a known allele are assigned the same allele number whereas those that differ from any known allele, even at a single nucleotide site, are assigned new allele numbers. The assignment of alleles at each locus is carried out using the software at the pneumococcal MLST web site. The alleles at each of the seven loci define the allelic profile of each isolate and their sequence type (ST). 
          </p>
          <p>
            Allelic profiles are shown as a series of seven integers which correspond to the alleles at each of the loci, in the order aroE, gdh, gki, recP, spi, xpt and ddl. The relatedness between the isolates is represented as a dendrogram, constructed by the unweighted pair-group method with arithmetic averages (UPGMA), from the matrix of pair-wise differences in the allelic profiles. The allelic profiles of the reference isolates of each clone are deposited within the pneumococcal database at the MLST web site.
          </p>
          <header>
            <h2>PBP fingerprinting:</h2>
          </header>
          <p>
            The pbp1a, pbp2b and pbp2x genes were amplified by PCR using methods and primers previously described (<a href="#4">4</a>). Amplified genes were digested with HaeIII+DdeI (pbp1a) and HaeIII+RsaI (pbp2b and pbp2x) and electrophoresed on 3% gels.
          </p>        
        </div><hr>
        <header>
            <h1><i class="fa fa-pencil lbrown"></i> Referenced</h1>
        </header>
        <div class="desc">
          <ul>
            <li>
              <p>
                <a name="1"></a>1. <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=8253973">Lef&egrave;vre, J. C., G.
    Faucon, A. M. Sicard, A. M. Gasc</a>. 1993. DNA fingerprinting of <em>Streptococcus Pneumoniae</em> strains by pulsed field gel electrophoresis. J. Clin. Micro. 31:2724-2728.
              </p>
            </li> 
            <li>
              <p>
                <a name="2"></a>2. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=8619583">McDougal, L. K., J. K.
    Rasheed, J. W. Biddle and F. C. Tenover</a>. 1995. Identification of multiple clones of extended-spectrum cephalosporin-resistant <i> Streptococcus pneumoniae</i> isolates in the United States. Antimicrob. Agents Chemother. 39:2282-2288.
              </p>
            </li>
            <li>
              <p>
                <a name="3"></a>3. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=9846740">Enright,
    M. C. and B. G. Spratt</a>. 1998. A multilocus sequence typing scheme for <i>Streptococcus
    pneumoniae</i>: identification of clones associated with serious invasive disease. Microbiology 144:3049-3060.
              </p>
            </li>
            <li>
              <p>
                <a name="4"></a>4. 
                <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=10608770">Gherardi,
    G., C.G. Whitney, R.R. Facklam and B. Beall.</a>&nbsp; 2000. Major related
    sets of antibiotic-resistant pneumococci in the United States as determined
    by pulsed-field gel electrophoresis and pbp1a-pbp2b-pbp2x-dhf restriction
    profiles. J. Infect. Dis. 181:216-229.
              </p>
            </li>
          </ul>    
        </div>
    </div>
  </section>


</div>

<footer>
      <!------------------ Social Communications ---------------->
      <section id="social-footer">
        <div class="container">
          <div class="row">
              <!-- Twitter widget here -->
              <div class="col-lg-12 col-md-12">
                  <h2><i class="fa fa-sitemap lbrown"></i> Site Map</h2>
                  <div class="copy">
                    <p>
                        <ul>
                          <li><a href="index.html">Home</a></li>
                          <li>
                            <a href="members.html">Members</a> / 
                            <a href="history.html" >History</a> /
                            <a href="members.html">Members</a>
                          </li>
                          <li>
                            <a href="criteria.html">Criteria & Nomenclature</a> / 
                            <a href="clone-collection.html">Clone Collection</a> / 
                            <a href="map_view.html" >Clone Map</a> / 
                            <a href="molecular-typing.html">Molecular Typing</a>                          
                          </li>
                          <li><a href="contact.html">Contact</a></li>
                    
                        </ul>

                        <span class="color">PMEN </span> &copy; 
                        <a href="#">2014</a>
                    </p>
                  </div>
              </div>
          </div>
      </div>
    </section>
    <div class="clearfix"></div>
</footer>
    <script src="js/vendor/modernizr-2.6.2.min.js"></script>
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
    <script type="text/javascript" src="http://code.jquery.com/jquery.min.js"></script>
    <script src="http://netdna.bootstrapcdn.com/bootstrap/3.1.0/js/bootstrap.min.js"></script>
    <script src="js/custom.js"></script>

</body>
</html>